rllib/tuned_examples/bc/cartpole_bc.py

```diff
@@ -75,7 +75,7 @@
 
 stop = {
     f"{EVALUATION_RESULTS}/{ENV_RUNNER_RESULTS}/{EPISODE_RETURN_MEAN}": 120.0,
-    TRAINING_ITERATION_TIMER: 350.0,
+    TRAINING_ITERATION_TIMER: 350,
```

Comment:
??
Why do we stop by the iteration timer? :) Oh, is this b/c of the Ray.Data-caused slowness per iteration still?